spyder/plugins/editor/widgets/recover.py

```diff
@@ -206,6 +202,13 @@ def add_cancel_button(self):
         button_box.rejected.connect(self.reject)
         self.layout.addWidget(button_box)
 
+    def center(self):
```

Comment:
@dalthviz, what happens when we try to center this and you have two monitors in your system?

Reply:
The dialog is shown centered in the first screen

Reply:
Maybe instead of `QApplication.desktop().screenGeometry(0)` we should use `QApplication.primaryScreen()` here @ccordoba12 ?

Reply:
`primaryScreen` is the screen where Spyder was launched?

Reply:
Probably not, it is what the system defines as "primary" so to know which screen the widget lives (The QMainWindow for example) we could look into:

https://stackoverflow.com/questions/49167204/find-screen-a-qwidget-is-located-on

Reply:
Probably not, it is what the system defines as "primary" so to know which screen the widget lives (The QMainWindow for example) we could look into:

https://stackoverflow.com/questions/49167204/find-screen-a-qwidget-is-located-on

Reply:
At the end this resulted in using the window settings from the config files. This is a preview:

![auto](https://user-images.githubusercontent.com/16781833/68444477-eff07900-01a4-11ea-9948-b52d966fb7a6.gif)